.github/workflows/release.yml

```diff
@@ -7,14 +7,18 @@ on:
 
 jobs:
   goreleaser:
+    permissions:
+      id-token: write
+      contents: write
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - run: git fetch --prune --unshallow
-      - uses: actions/setup-go@v1
+      - uses: actions/setup-go@v2
         with:
           go-version: 1.17.x
-      - uses: goreleaser/goreleaser-action@v1
+      - uses: sigstore/cosign-installer@v1.3.1
```

Comment:
Please update to cosign-installer@v1.4.1

Reply:
thank you 🙋🏻‍♂️